posthog/queries/properties_timeline/properties_timeline.py

```diff
@@ -0,0 +1,96 @@
+import json
+from typing import Any, Dict, List, TypedDict
+
+from posthog.models.filters.properties_timeline_filter import PropertiesTimelineFilter
+from posthog.models.person.person import Person
+from posthog.models.property.util import extract_tables_and_properties, get_single_or_multi_property_string_expr
+from posthog.models.team.team import Team
+from posthog.queries.insight import insight_sync_execute
+
+from .properties_timeline_event_query import PropertiesTimelineEventQuery
+
+
+class PropertiesTimelinePoint(TypedDict):
+    timestamp: str
+    properties: Dict[str, Any]
+    relevant_events_since_previous_point: int
+
+
+PROPERTIES_TIMELINE_SQL = """
```

Comment:
I don't understand this query (probably because I'm lacking context on what problem `relevant_events_since_previous_point` solves.

- Have you tried this query in production for large teams?
- Are there other ways to solve the same problem (e.g. by doing multiple queries)

Reply:
`relevant_events_since_previous_point` will be shown in the timeline to inform users how many events happened between the points in time shown (since this is directly relevant to most insights).

Reply:
Reworked this to `relevant_event_count`, which is more relevant (events for _this_ point instead of _for previous_ point).